tests/integration/cmd_push_test.go

```diff
@@ -32,13 +32,13 @@ var _ = Describe("odo push command tests", func() {
 
 	Context("Check pod timeout", func() {
 
-		It("Check that pod timeout works and we time out immediately..", func() {
-			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
-			helper.CmdShouldPass("odo", "component", "create", "--s2i", "nodejs", cmpName, "--project", commonVar.Project, "--context", commonVar.Context, "--app", appName)
-			helper.CmdShouldPass("odo", "preference", "set", "PushTimeout", "1")
-			output := helper.CmdShouldFail("odo", "push", "--context", commonVar.Context)
-			Expect(output).To(ContainSubstring("waited 1s but couldn't find running pod matching selector"))
-		})
+		// TODO: issue with PushTimeout
```

Comment:
I assume this has something to do with the PushTimeout ping on chat. Rest of the code looks good to me.